lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.html

```diff
@@ -45,7 +45,7 @@
     <adf-toolbar>
         <adf-toolbar-title>
             <ng-container *ngIf="!showBreadcrumbs()">
-                <span role="heading" aria-level="3" class="adf-search-results-label">{{ 'NODE_SELECTOR.SEARCH_RESULTS' | translate }}</span>
+                <h2 class="adf-search-results-label">{{ 'NODE_SELECTOR.SEARCH_RESULTS' | translate }}</h2>
```

Comment:
Is there any h1 tag above this one? Otherwise this will raise new accessibility issue, h tags should be placed in order

Reply:
I see content-node-selector-panel is used in three places: 
1.
<img width="1728" alt="Screen Shot 2022-11-08 at 15 37 52" src="https://user-images.githubusercontent.com/115619721/200594438-8e0c6118-79bc-4e9b-b7ae-07538ad4e0e1.png">
2.
<img width="1728" alt="Screen Shot 2022-11-08 at 15 38 10" src="https://user-images.githubusercontent.com/115619721/200594760-2d3d5f1c-2d54-4715-84de-6dce2cc4a7b0.png">

Both have h1 tag in template itself. 

I see it is used also in third place which is AttachFileWidgetDialogComponent but can't find usage of that component to ensure there is any h1 tag. Do you have idea where in application it is used?

Reply:
ok I will try to check that one place too to ensure it is working there as well. Rest places have h1 tag like you can see on screenshots :) Will let you know about third place when I check. If you know any other edge cases where we use that component then let me know but what I can see except these 3 places there is no more usage.